client/styles/abstracts/_variables.scss

```diff
@@ -110,3 +110,13 @@ $console-error-color: #ff5f52;
 
 $toast-background-color: #979797;
 $toast-text-color: $white;
+
+$form-title-color: rgba(51, 51, 51, 0.87);
```

Comment:
I think it would make sense to add all of these colors to the big color map. Or maybe it doesn't since maybe it doesn't make sense to have these forms have different themes. We'll have to discuss this one.